src/rpc.ts

```diff
@@ -35,15 +35,17 @@ router.post('/', async (req, res) => {
       }
 
       if (verifySubscribe(params.email, params.address, params.signature)) {
-        await subscribe(params.email, params.address);
-        queueSubscribe(params.email, params.address);
+        const result = await subscribe(params.email, params.address);
```

Comment:
much readable it will be if this will return an object with property you want to check:
```ts
const { wasInserted } = await subscribe(params.email, params.address);

if (wasInserted) { /*...*/ }
```